pkg/minikube/assets/vm_assets.go

```diff
@@ -104,6 +111,15 @@ func (f *FileAsset) GetLength() (flen int) {
 	return int(fi.Size())
 }
 
+// GetModTime returns modification time of the file
+func (f *FileAsset) GetModTime() time.Time {
+	fi, err := os.Stat(f.AssetName)
+	if err != nil {
+		return time.Time{}
```

Comment:
I think this should return the error, since it will be impossible otherwise for the caller to determine if there is an error, or has an mtime of epoch.

Reply:
done